doc/input-output-reference/src/overview/group-radiative-convective-units.tex

```diff
@@ -1595,9 +1607,15 @@ \subsubsection{Inputs}\label{inputs-7-019}
   OutdoorDryBulbTemperature
 \item
   OutdoorWetBulbTemperature
+\item
+  SurfaceFaceTemperature
+\item
+  SurfaceInteriorTemperature
 \end{itemize}
 
-Operative temperature for radiant system controls is the average of Mean Air Temperature and Mean Radiant Temperature. If the user does not select a control type, \textbf{MeanAirTemperature} control is assumed by EnergyPlus. See the throttling range and control temperature schedule fields below for more information.
+\emph{Operative temperature} for radiant system controls is the average of Mean Air Temperature and Mean Radiant Temperature.  The \emph{SurfaceFaceTemperature} option allows the user to control the radiant system using the inside face surface temperature of the radiant system (the inside surface temperature).  The \emph{SurfaceInteriorTemperature} option will allow the user to control the radiant system using a surface temperature that is calculated inside the radiant system.  This point will be defined by the  \hyperref[constructioninternalsource]{Construction:InternalSource} description for the system.  In that input, the user has the option to calculate a temperature at a particular point in the construction.  The radiant system will then use this information for controlling the slab not just producing temperatures for outputting at that point.  Users should consult the input for Field: Temperature Calculation Requested After Layer Number for more information.  Note that for the SurfaceFaceTemperature and SurfaceInteriorTemperature the surface being used for control is the surface listed above in the field for Surface Name.  If the user enters a group of surfaces for that input, the \emph{first} surface in the radiant group is the surface for control purposes.
+
+If the user does not select a control type, \textbf{MeanAirTemperature} control is assumed by EnergyPlus. See the throttling range and control temperature schedule fields below for more information on how the setpoint temperature is established for this particular radiant system.
```

Comment:
I'm not going to dwell over the doc changes for now.  I'd just say in the future, if you are making a cleanup pass on the code in a first branch, don't worry about including any doc changes so that the doc changes go in at the exact same time as the functional code changes.